src/lib/y2users/parsers/passwd.rb

```diff
@@ -47,6 +48,7 @@ def parse(content)
           user.shell = values[PASSWD_MAPPING["shell"]]
           user.gecos = values[PASSWD_MAPPING["gecos"]].to_s.split(",")
           user.home =  values[PASSWD_MAPPING["home"]]
+          user.authorized_keys = Yast::Users::SSHAuthorizedKeyring.new(user.home).read_keys
```

Comment:
Umm, the authorized keys is not a passwd field. I would move this to another place, maybe reader?